### DISCUSSION TITLE
Installation caribou

### DISCUSSION THREAD

**Discussion by gerd0355** [cariboulabs/cariboulite#44] [General]
I have got my caribou yesterday.
Now I wanted to install the software on Raspberry Zero as described in cariboulabs/cariboulite  Readme.md . In Step4 I cannot find the script setup.sh!
What is wrong?
Regards 
Gerhard

**Comment by ImDroided** [comment]
everything is broken. go to the dev branch R1 and use the install script but even that is broken. You will have to fix all the stuff it fails on and continue till its completed everything. took me a while. The developer apparently isnt ready for us yet. It is being worked on but its currently a cluster........

**Reply by ImDroided** [reply]
The setup.sh doesnt even exist.

**Reply by ryancnelson** [reply]
the good stuff is apparently in git branch `develop_R1`

`git checkout develop_R1`
```

root@raspberrypi:/home/ryan/cariboulite# git checkout develop_R1
Updating files: 100% (482/482), done.
Branch 'develop_R1' set up to track remote branch 'develop_R1' from 'origin'.
Switched to a new branch 'develop_R1'
root@raspberrypi:/home/ryan/cariboulite# ls -la install.sh 
-rwxr-xr-x 1 root root 4452 Feb  8 21:27 install.sh

```

ta-da!

**Reply by gerd0355** [reply]
Hi Ryan,
thank you for your quick answer. I found the install.sh file, copied it to my raspberry and executed the file. I had no errors. Because I have some experience in gnuradio I tried to test the grc-examples. I had no success. I think the soapy-apis are not correct. 
So I tried  to compile the SoapySDR API as described in the README.md from CaribouLite. I had no success, because I could not make the code (Error: zmq.h not found). It should be in ..../software/libcariboulite/src/production_utils .
Can you help me please.
By the way: When I tried to compile the Soapy APIs I had problems to find the path way for the files . 
Best regards
Gerhard
